pkg/conf/conf.go

```diff
@@ -156,6 +156,24 @@ type Main struct {
 	// Histogram parameters for the processing duration
 	ProcessingDurationBucketFactor float64
 	ProcessingDurationBuckets      int
+
+	// RateLimiter config
+	// Limits the number of records in the time window a container can send to maximum of
+	// RateLimiterContainerRecordLimit records.
+	RateLimiterContainerRecordLimit int
```

Comment:
Same as for the next parameter: I think it's worth mentioning that this enables per-container rate limiter.

Reply:
Right. Added.